7-gce/gce/deployment_manager/config.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+imports:
+- name: bookshelf.jinja
+  path: ./bookshelf.jinja
+
+resources:
+- name: bookshelf
+  type: bookshelf.jinja
+  properties:
+    zone: us-central1-f
+    machine-type: n1-standard-1
+    machine-image: https://www.googleapis.com/compute/v1/projects/debian-cloud/global/images/debian-8-jessie-v20151104
```

Comment:
Gave me a 503, I have a bug for it, not sure whether image families are supposed to be supported, I asked their list